rootToH5converter/Event.py

```diff
@@ -105,7 +121,8 @@ def fill_jets(self, use_fat_jets=False):
             return
             
         prefix = "Fat" if use_fat_jets else ""
-        
+        AKx = "AK8" if use_fat_jets else "AK4"
```

Comment:
more descriptive (and starting with lower case) name for "AKx"?

Reply:
I can change that